app/scripts/helpers/StaticHelpers.js

```diff
@@ -52,6 +52,14 @@ define(function(){
 
         capitalizeFirstLetter: function(string) {
             return string.charAt(0).toUpperCase() + string.slice(1);
+        },
+
+        createFace: function(data){
+            return new THREE.Face3( data[0], data[1], data[2],
+                [ new THREE.Vector3( data[3][0], data[3][1], data[3][2] ),
+                  new THREE.Vector3( data[3][3], data[3][4], data[3][5] ),
+                  new THREE.Vector3( data[3][6], data[3][7], data[3][8] )
+                ]);
```

Comment:
Thanks @pboyer for looking. Agree that we should create some kind of ThreeUtils class with a dependency on ThreeJS